### PR TITLE
initial documentation

### PR DESCRIPTION
**Pull request by Hussein-Mahfouz**
Start on #62

### CODE REVIEW

File: data/external/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+`nts/`
+
+UKDA-5340-tab:
+- Download the UKDA-5340-tab from the UK Data Service [here](https://beta.ukdataservice.ac.uk/datacatalogue/studies/study?id=5340)
+- Step 1: Create an account
+- Step 2: Create a project and request access to the data
+    - We use the `National Travel Survey, 2002-2023` dataset (SN: 5340)
+- Step 3: Download TAB file format
+
+`travel_times/`
```
Comment (sgreenbury) [inline data/external/README.md:103]: As discussed, add information on dataframe structure (i.e. expected columns)

File: data/external/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        ```
+        and replace `west-yorkshire` and `2020` with your preferred option.
+
+`boundaries/`
+
+- MSOA_DEC_2021_EW_NC_v3.geojson
+- oa_england.geojson
+- study_area_zones.geojson
+
+`centroids/`
```
Comment (sgreenbury) [inline data/external/README.md:89]: As discussed, add data source

File: data/external/README.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        ```shell
+        cargo run --release -- \
+            --rng-seed 0 \
+            --flat-output \
+            --year 2020 \
+            config/England/west-yorkshire.txt
+        ```
+        and replace `west-yorkshire` and `2020` with your preferred option.
+
+`boundaries/`
```
Comment (sgreenbury) [inline data/external/README.md:83]: As discussed, add data source

File: README.md [hunk 4]
```diff
@@ -19,15 +51,140 @@ cd acbm
...
+│   │   │   │   ├── <region>_info_per_msoa.json
+│   │   │   │   ├── <region>.pb
+│   │   │   │   ├── <region>_people.parquet
+│   │   │   │   ├── <region>_time_use_diaries.parquet
+│   │   │   │   ├── <region>_venues.parquet
+│   │   │   │   ├── README.md
+│   ├── interim
+│   │   ├── assigning (Generated in Script 3)
+│   │   └── matching (Generated in Script 2)
+│   └── processed
```
Comment (sgreenbury) [inline README.md:112]: As discussed, I'll aim to update the notebooks and scripts to read/output files to match this structure.

File: data/external/README.md (hunk 2 above)
Comment (Hussein-Mahfouz) [reply data/external/README.md:90]: done in https://github.com/Urban-Analytics-Technology-Platform/acbm/pull/63/commits/2ca4d10d7ed6cfb4dfb9a038f08c5f9877ce8323

File: data/external/README.md (hunk 3 above)
Comment (Hussein-Mahfouz) [reply data/external/README.md:84]: done in https://github.com/Urban-Analytics-Technology-Platform/acbm/pull/63/commits/2ca4d10d7ed6cfb4dfb9a038f08c5f9877ce8323

File: data/external/README.md (hunk 1 above)
Comment (Hussein-Mahfouz) [reply data/external/README.md:104]: done in https://github.com/Urban-Analytics-Technology-Platform/acbm/pull/63/commits/2ca4d10d7ed6cfb4dfb9a038f08c5f9877ce8323

**Comment by Hussein-Mahfouz** [comment]
I've updated the `config/README` and the `data/external/README`. This is ready to merge after implementation of #53. I will merge it Tuesday so that it is ready before the presentation

**Comment by Hussein-Mahfouz** [comment]
Merging now, #53 can be in a different PR

**Pull request merged**